src/SimpleSelect.js

```diff
@@ -72,6 +72,7 @@
         highlightedUid: highlightedUid,
         onHighlightedUidChange: onHighlightedUidChange,
         inputProps: inputProps,
+        listboxId: this.props.listboxId,
```

Comment:
can this be removed now?

Reply:
yep, since we're just generating uuid inside ReactSelectize.js, I no longer need this and it's been removed.